pages/data-providers/[data-provider-id]/deposit-compliance.module.css

```
@@ -15,12 +15,24 @@
   }
 }
 
-.compliance-level {
+.data-overview {
   flex: 2 2 60%;
   width: 100%;
 }
 
+.numbers {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 1rem;
```

Fixes extra spacing below:
```suggestion
  margin: 0 calc(-1 * var(--component-padding-x, 1rem));
```

---

I am not sure what you tried to achieve with this rule. It moves numbers to the left.

---

The card has padding, let's say `16px`. I want to create 2 columns inside with 32px spacing between them.

To do so, I create a _row_ with a negative left and right margin (`-16px`) and 2 _columns_ inside the _row_ with `16px` left and right margin/padding.

It's a more flexible solution than using `:first-child` and `:last-child` with removing margin because of columns may break to the next row.

---

it doesn't work when you make flex basis 50% and use wrap

---

Yeah, the flex-basis should be relatively small. If it works with 1px, it can be so; otherwise any small amount that fits the number, e.g. 10rem.